docs/source/distributions/kubernetes_deployment.md

```diff
@@ -33,7 +38,8 @@ metadata:
   name: hf-token-secret
 type: Opaque
 data:
-  token: $(HF_TOKEN)
+  token: $(echo -n "$HF_TOKEN" | base64)
```

Comment:
```suggestion
  token: $HF_TOKEN
```
I think we can just fix the substitution here and do all the encoding in the export?